libmscore/style.cpp

```diff
@@ -940,7 +940,7 @@ static const StyleType styleTypes[] {
       { Sid::footerFontStyle,               "footerFontStyle",              int(FontStyle::Normal) },
       { Sid::footerColor,                   "footerColor",                  QColor(0, 0, 0, 255) },
       { Sid::footerAlign,                   "footerAlign",                  QVariant::fromValue(Align::LEFT) },
-      { Sid::footerOffset,                  "footerOffset",                 QPointF() },
+      { Sid::footerOffset,                  "footerOffset",                 QPointF(0.0, 5.0)        },
```

Comment:
This one line change seems to be the only one that is needed in this PR to fix the issue at hand, isn't it?

Reply:
yes sir. Because only 5mm value was required to be added to the y offset of footer. And a lot of bugs were related to this.

Reply:
Though , I am extremely embarrassed that the space is still being missing and this was the not the way to solve it. Sorry. I will try to figure out some other ways or things. So that travis also doesn’t fails.

Reply:
Here the space is not missing. But you need to get rid of the other 2 changes